IntegrationTests/SPMIntegration/SPMIntegrationTests/SPMIntegrationTests.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  SPMIntegrationTests.swift
+//  SPMIntegrationTests
+//
+//  Created by Andrés Boedo on 10/28/20.
+//
+
+import XCTest
+@testable import SPMIntegration
+import Purchases
+
+class SPMIntegrationTests: XCTestCase {
+
+    func testCanConfigureTheSDK() throws {
+        RCIntegrationRunner().start()
+    }
+
+    func testCanFetchPurchaserInfo() throws {
```

Comment:
I ended up repeating this logic for each integration. Not a huge fan of doing that. 
I'm not sure how best to solve it, though - the issue is that they have to import the app's target, which they import by name. 
And they're all named differently. 

Thoughts: 
- I could unify the target name for all of them, then you have the same file always
- I could use `sed` to replace the target name. That works, but is a bit annoying when developing, because you'd have to update it manually every time you're building locally.

Reply:
Could we have a swift file in the `CommonFiles` with a function we could execute in these tests?

Reply:
yeah, that's the idea of the RCIntegrationRunner class. But I couldn't find a way to unify further using that approach

Reply:
I see. I was thinking on basically delegating the whole test. Like delegating the asserts to another class. Like literally moving the `testCanFetchPurchaserInfo` to RCIntegrationRunner and having all the integration tests just call `RCIntegrationRunner.testCanFetchPurchaserInfo()`. It doesn't have to be in RCIntegrationRunner.

Reply:
🤔 I guess it could work if we have the apps import XCTest, I'll give it a shot

Reply:
I've been trying it out but it's not ideal - I'd need for the app to import XCTest and to include a unit test, essentially. 
I thought of another way of doing it that might be better, though: adding a single compiler flag to each of the projects (carthage, cocoapods, xcodeIntegration, spm) and then using that to make the correct import call. 
That way we can have a single, common unit test file. And it's easy to add others.

Reply:
I'm working on that now, will update soon

Reply:
@vegaro updated: https://github.com/RevenueCat/purchases-ios/pull/381/files/4446385534a9142821517f7ea6c30930ee2f4b77..43ade17af01c4b436f83532d53dc45b6f5df5920
- unified all of the test files into one
- added compiler flag to each project
- used it to selectively import the correct target.